nw30typescriptExample/nightwatch/commands/problemCommand.ts

```diff
@@ -2,7 +2,7 @@ import { NightwatchClient, NightwatchExpectResult } from 'nightwatch';
 
 export default class WaitForLoadScreen {
   async command(this: NightwatchClient, expectedTitle: string) {
-    browser.waitUntil(async () => {
+    await browser.waitUntil(async () => {
```

Comment:
I tried that too, but it acts the same. Even if I also add an await on this command in the test itself.

Reply:
I also encountered some issues with waitUntil which we need fixed, but eventually I found a format that works. I used it in some of our plugins, e.g. here: https://github.com/nightwatchjs/vite-plugin-nightwatch/blob/main/nightwatch/commands/mountReactComponent.js#L121